File: game-admin/src/main/java/com/dkm/admin/entities/AdminUserEntity.java
package com.dkm.admin.entities;

import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

@Data
@TableName("tb_admin_users")
public class AdminUserEntity {
   @TableId
   private  int id;
   private long userId;

}

File: game-admin/src/main/java/com/dkm/admin/entities/AdminUserEntity.kt
package com.dkm.admin.entities

import com.baomidou.mybatisplus.annotation.TableId
import com.baomidou.mybatisplus.annotation.TableName

@TableName("tb_admin_users")
data class AdminUserEntity(
        @TableId
        val id: Int,
        val userId: Long
)